apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js

```diff
@@ -135,17 +135,22 @@ const ReviewApplication = ({ issue, requestIndex }) => {
 
           <FieldTitle>Application Status</FieldTitle>
           <div css="margin: 10px 0">
-            {request.review.approved ? (
-              <div css="display: flex; align-items: center">
-                <IconCheck color={`${theme.positive}`} css="margin-top: -4px; margin-right: 8px"/>
-                <Text color={`${theme.positive}`}>Accepted</Text>
-              </div>
-            ) : (
-              <div css="display: flex; align-items: center">
-                <IconCross color={`${theme.negative}`} css="margin-top: -4px; margin-right: 8px" />
-                <Text color={`${theme.negative}`}>Rejected</Text>
-              </div>
-            )}
+            <div css="display: flex; align-items: center">
+              {request.review.approved ? (
+                <>
+                  <IconCheck color={`${theme.positive}`} css="margin-top: -4px; margin-right: 8px"/>
+                  <Text color={`${theme.positive}`}>Accepted</Text>
+                </>
+              ) : (
+                <>
+                  <IconCross color={`${theme.negative}`} css="margin-top: -4px; margin-right: 8px" />
+                  <Text color={`${theme.negative}`}>Rejected</Text>
+                </>
+              )}
+              <Text color={`${theme.contentSecondary}`} css={`margin-left: ${GU}px`}>
+                {formatDate(new Date(request.review.reviewDate), 'd MMM yy, h:MM a (z)')}
+              </Text>
+            </div>
```

Comment:
I think we could clean this up a bit by creating a new `Status` component.

```js
const statuses = [
  { color: 'negative', text: 'Rejected' },
  { color: 'positive', text: 'Accepted' },
]

const Status = ({ review }) => {
  const theme = useTheme()
  const status = statuses[Number(review.approved)]
  const text = status.text
  const color = theme[status.color]
  return (
    <div css="display: flex; align-items: center">
      <IconCheck color={color} css={`margin-top: -${0.5 * GU}px; margin-right: ${GU}px`} />
      <Text color={color}>{text}</Text>
      <Text color={`${theme.contentSecondary}`} css={`margin-left: ${GU}px`}>
        {formatDate(new Date(review.reviewDate), 'd MMM yy, h:MM a (z)')}
      </Text>
    </div>
  )
}
```

Reply:
Done in PR #1698

Reply:
Do these two PRs have conflicts, then? I'm not sure I understand the intended workflow.